index.html

```diff
@@ -23,67 +24,79 @@
           <div class="navbar__logo">
             <a href="#" class="logo link fz-18 fw-700 1h-20">Daphine</a>
           </div>
-          <div class="humberger menu"></div>
-          <div class="humberger">
-            <ul>
+          <img
+            id="humberger"
+            class="humberger-toggle"
+            src="./humberger.svg"
+            alt="menu icon"
+          />
+          <img src="./assets/cancel.svg" class="cancel hide" alt="cancel" />
+          <div class="menu-icon menu-toggle">
```

Comment:
- [ ] Currently, our mobile menu is showing automatically when on desktop view let's make sure we get rid of that
![image](https://user-images.githubusercontent.com/87689547/157855040-18f10fe3-e325-4cf9-82c8-2b2e42da5727.png)

Reply:
* Issue resolved